releasenotes/v1.8.4.md

```diff
@@ -0,0 +1,19 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v1.8.3
+
+## :bug: Bug Fixes
+
+- bump golang to 1.23.5 (#871)
+
+## :seedling: Others
+
+- Bump CAPI to v1.8.9 (#867)
+- Bump the k8s.io/* deps to v0.30.9 (#860)
+- Bump sigs.k8s.io/controller-runtime from 0.18.6 to 0.18.7 (#854)
+- bump x/net to v0.34.0 (#850)
+
+## :recycle: Superseded or Reverted
```

Comment:
if we dont have any PR here, do we still need the section title ?

Reply:
/hold
true